RoT/Worldmap/schinese/worldmap.tra

```diff
@@ -0,0 +1,41 @@
+
+@ROT_785001 = "阿拉克斯"
```

Comment:
@Lzw104522773 
Also I am pretty sure that this file should be encoded in your local language encoding.